doc/api/projections_api.rst

```diff
@@ -6,16 +6,13 @@
    :members:
    :show-inheritance:
 
-``matplotlib.projections.polar``
-================================
+Built in projections
```

Comment:
```suggestion
Built-in projections
```

Reply:
Everything I've found online says this should indeed have a dash because it is a compound adjective.  https://www.merriam-webster.com/dictionary/built-in

Reply:
Ooos sorry I was misreading the diffs.  Looks like you took this change.